lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -270,10 +362,46 @@ rule MODELType2WebHome {
 		name <- 'type',
 		type <- 'StaticList',
 		value <- s.modeltype.mapStandardModelType
+	),
+	--begin documentation
+	mc1:XWIKI!Object (
```

Comment:
Same here, `mc` is the only instance that should exist of the object `LPCode.ModelClass`.  Just add the property `documentation`.